cs1/graphics.py

```diff
@@ -101,13 +107,15 @@ def set_line_thickness(thickness):
   global _fg
   _check()
   _fg.line_width = thickness
+  _tb.put_token()
```

Comment:
Here and below: only functions that issue draw commands need to be rate limited.